src/plugin/morphologyHDF5.cpp

```diff
@@ -213,35 +213,53 @@ void MorphologyHDF5::_readPoints(unsigned int firstSectionOffset)
                 "Reading morphology file '" + _file->getName() + "': bad number of dimensions in 'points' dataspace"));
         }
         std::vector<std::vector<float>> vec(dims[0]);
-        // points.resize(dims[0]);
         dataset.read(vec);
-        bool noNeurites = (firstSectionOffset == -1);
```

Comment:
@arsenius7 the original version tested an *`unsigned int`* for a negative value here. Hence me setting it to `int`.

Reply:
That I understood (see also my comment about unit test below).
Btw, wasn't there a compilation warning indicating this comparison?

Reply:
No, that would require some additional compiler flags. I added  those locally just now, need to find a more minimal set than my standard one…

Reply:
@arsenius7 there you go :upside_down_face: 

The old `unsigned int` does not compile now with the new CMake options.